src/AstToRamTranslator.cpp

```diff
@@ -747,7 +747,6 @@ std::unique_ptr<RamStatement> AstToRamTranslator::ClauseTranslator::translateCla
                             "Unsupported complex aggregation body encountered!");
                 }
             }
-
```

Comment:
Try to avoid making unrelated changes like this. It makes the PRs slower and more confusing to review - and in this case isn't needed.